btk/draw_blends.py

```diff
@@ -243,9 +243,17 @@ def __next__(self):
             batch_blend_cat[s.name] = []
             batch_obs_cond[s.name] = []
             image_shape = (
-                len(s.filters),
-                pix_stamp_size,
-                pix_stamp_size,
+                (
+                    len(s.filters),
+                    pix_stamp_size,
+                    pix_stamp_size,
+                )
+                if not self.channels_last
+                else (
+                    pix_stamp_size,
+                    pix_stamp_size,
+                    len(s.filters),
+                )
```

Comment:
maybe avoid the command taking up several lines by declaring the variables separately? e.g.
```
shape1 = (len(s.filters), pix_sample_size, pix_stamp_size)
shape2 = pix_sample_size, pix_stamp_size, (len(s.filters))
image_shape = shape1 if not self.channels_last else shape2
```

Reply:
did this in the latest commit